public/app/plugins/datasource/loki/add_label_to_query.ts

```diff
@@ -39,22 +39,36 @@ export function addLabelToQuery(query: string, key: string, operator: string, va
   }
 }
 
-type StreamSelectorPosition = { from: number; to: number; query: LokiVisualQuery };
-type PipelineStagePosition = { from: number; to: number };
```

Comment:
I have unified these into `type Position = { from: number; to: number };`. We don't need query here, as it can be retrieved directly in the function that uses `StreamSelectorPosition`.